articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+
+## Step 4. Set up target environment
+
+In Public Azure, select and verify target resources.
+
+1. In **Prepare infrastructure** > **Target**, select the Azure subscription you want to use.
+
+2. Specify the target deployment model.
+
+3. Site Recovery will check to see if you have a compatible Public Azure Storage account and network. If you don't you will need to create them. This can be done with the Storage **Account** and **Network** buttons under Target.
+
+4. Select **OK**.
+
+    ![Azure site recovery infrastructure 1](images/azs-browser-recovery-infrastructure-configuration3.png)
+
+## Step 5: Enable replication
+
+1. Select the final tab **Replication Settings**.
+
+2. Select **Create and Associate** and specify a policy name.
+
+3. In **RPO threshold**, specify the recovery point objective (RPO) limit.
+
+    * This setting does not affect replication, which is continuous. It simply issues an alert if the threshold limit is reached without a recovery point being created.
+
+4. In **Recovery point retention**, Specify how long each recovery point is kept. Replicated VMs can be recovered to any point in the specified time window.
+
+5. In **App-consistent snapshot frequency**, specify how often application-consistent snapshots are created.
+
+    * A app-consistent snapshot is a point-in-time snapshot of the app data inside the VM.
+
+    * Volume Shadow Copy Service (VSS) ensures that apps on the VM are in a consistent state when the snapshot is taken.
+
+6. Select **OK** to create the policy.
+
+    ![Azure site recovery infrastructure 2](images/azs-browser-recovery-infrastructure-configuration2.png)
+
+### Adding admin account
+
+After the setup has complete if you navigate to the configuration servers desktop you will find a tool called **Cspconfigtool**, run this tool and navigate to the **Manage Accounts** tab, select **Add Account** and then fill out the details matching the admin accounts details on the VM you wish to replicate. It may take a while for the configuration server to sync with Public Azure
+
+![Azure site recovery account manager](images/azs-siterecovery-account-manager.png)
+
+### Enable replication
+
+Make sure you've completed all the tasks in **Step 1: Preparation of VMs**. On Public Azure then enable replication as follows:
+
+1. Under **Protected items** in the vault select **Replicated items**.
+
+2. Select **Replicate**.
+
+3. In Source, select **On-premises**.
+
+4. In **Source location** select the configuration server you set up.
+
+5. In Machine type, select Physical machines.
+
+6. Select the process server (configuration server).
+
+    ![Azure replication source](images/azs-browser-replication-source.png)
+
+7. In Target, select the **subscription** and the **Post-failover resource group** in which you want to create the VMs after failover. Choose the **Post-failover deployment model** that you want to use for the failed-over VMs.
```

Comment:
In the *Target* blade, select the **Subscription**...

(uppercase S for Subscription)